gulpfile.mjs

```diff
@@ -1959,6 +1959,16 @@ gulp.task("lint", function (done) {
     styleLintOptions.push("--fix");
   }
 
+  const prettierOptions = [
+    "node_modules/prettier/bin/prettier.cjs",
+    "**/*.json",
+    "--log-level",
+    "error",
```

Comment:
This doesn't seem to work to suppress output, since both locally and in GitHub Actions this causes a lot of undesirable console-spam.

Reply:
Is that option broken somehow, since the updated patch doesn't seem to help at all :-(

(I don't think we can/should land this unless we can get rid of all that "spamming".)

Reply:
I've just tested this patch (again) locally, by checking it out and running `gulp lint`, and unfortunately I can still reproduce the "spamming".

Reply:
My bad, I was testing with `gulp lint --fix`, I'll fix that.